### DISCUSSION TITLE
Environment variables type with Next.js

### DISCUSSION THREAD

**Discussion by CodeWorm00** [vercel/community#1195] [Help] [answered]
### Summary

Which environment variables uses React? Ones accessed from process.env, correct?

But which environment variables uses Next.js? Ones accessed from printenv command from OS terminal?

**Comment by trendy0413** [comment, marked as answer]
Environment Variables don't use React or Next.js to be exact.

For more information on using Environment Variables with Next.js → https://nextjs.org/docs/basic-features/environment-variables.

**Reply by CodeWorm00** [reply]
Thanks, appreciate it
